cityofaustin/atd-airflow
# DAG to execute AFD and EMS daily imports for VZ

## Intent & Associated issues & PRs

This PR has a sibling PR in the airflow repo, [#1241](https://github.com/cityofaustin/atd-vz-data/pull/1241).

This PR intends to close https://github.com/cityofaustin/atd-data-tech/issues/11730 in conjunction with https://github.com/cityofaustin/atd-vz-data/pull/1241. 

## Testing

I'm borrowing some from the [CRIS import DAG PR](https://github.com/cityofaustin/atd-airflow/pull/108).

## Testing

We have a few options here. This can be a review the code and rely on my testing, or I would be more than happy to give a demonstration of this working on my local VZ & airflow stacks. I can do that individually or maybe even as a dev-sync share if that would be helpful. 

Of course, individual testing is always welcome! Here is the general outline of how to do it. 

* Spin up the local VZ DB and populate it with data from the read replica.
* Bring up the local airflow stack on this branch. 
* Spin up the airflow stack
* You should be able to trigger the DAGs and see them work.
